v0/destinations/google_ads_enhanced_conversion/data/trackConfig.json

```diff
@@ -0,0 +1,129 @@
+[
+  {
+    "destKey": "partialFailure",
+    "sourceKeys": "properties.partialFailure",
+    "required": true
+  },
+  {
+    "destKey": "conversionAdjustments[0].gclidDateTimePair.gclid",
+    "sourceKeys": "properties.gclid",
+    "required": false,
+    "metadata": {
+      "type": "toString"
+    }
+  },
+  {
+    "destKey": "conversionAdjustments[0].gclidDateTimePair.conversionDateTime",
+    "sourceKeys": "properties.conversionDateTime",
```

Comment:
Lets rephrase this to `conversionTime`

Reply:
I am using this name for parity with the destination name.